File: Happnin.Data/Category.cs
using System.Collections.Generic;

namespace Happnin.Data
{
    public class Category : EntityBase
    {

        public CategoryTypes CategoryType { get; set; }
        public List<Event> Events { get; } = new List<Event>();

        public Category(CategoryTypes category) : this()
        {
            CategoryType = category;
        }

        private Category() { }
    }

    public enum CategoryTypes
    {
        Music = 0,
        Festival = 1,
        Comedy = 2,
        Culture = 3,
        Other = 4,
        Product = 5
    }
}

File: Happnin/ClientApp/src/shared/Category.js

export function Category(categoryId) {
    switch(categoryId){
        case 0:
            return 'Music'
        case 1:  
            return 'Festival'
        case 2: 
            return 'Comedy'
        case 3: 
            return 'Culture'
        case 4: 
            return 'Other'
        case 5: 
            return 'Product'
        default:
            return 'Other'
    }
};